src/MoBi.Presentation/Tasks/PKSimStarter.cs

```diff
@@ -80,9 +83,15 @@ public IBuildingBlock CreateIndividual()
          return _cloneManager.CloneBuildingBlock(buildingBlock) as TBuildingBlock;
       }
 
-      private object executeMethod(MethodInfo method)
+      public List<ExpressionParameterValueUpdate> UpdateExpressionProfileFromDatabase(ExpressionProfileBuildingBlock expressionProfile)
+      {
+         loadPKSimAssembly();
+         return executeMethod(getMethod(PKSIM_UI_STARTER_EXPRESSION_PROFILE_CREATOR, GET_EXPRESSION_DATABASE_QUERY), new object[] { expressionProfile }) as List<ExpressionParameterValueUpdate>;
+      }
+
+      private object executeMethod(MethodInfo method, object[] parameters = null)
       {
-         return method.Invoke(null, null);
+         return method.Invoke(null, parameters);
```

Comment:
lol

Reply:
Prior to this feature, we haven't had to send anything over the fence to MoBi. We used to send IShell, but we found out that wasn't necessary.